### PR TITLE
Add tests mantissa refactor

### PR DESCRIPTION
**Pull request by coreyar**
Before updating the Venus Core subgraph I added tests. I used the Isolated Pool tests as a base and updated the handlers and graph keys to pass the tests. 

This PR also changes how we use the vTokenDecimal. It is not assumed to be 8 but rather fetched from the contract. 
This change and correcting some values labeled as mantissa but are actually scaled down will need to be brought to the isolated pools subgraph

### CODE REVIEW

File: subgraphs/venus/schema.graphql [hunk 1]
```diff
@@ -20,27 +20,27 @@ Market stores all high level variables for a vToken market
 type Market @entity {
     #Fields that match Venus API
     "Borrow rate per block"
-    borrowRate: BigDecimal!
+    borrowRateMantissa: BigInt!
     "The vToken contract balance of BEP20 or BNB"
     cash: BigDecimal!
     "Collateral factor determining how much one can borrow"
     collateralFactor: BigDecimal!
```
Comment (gleiser-oliveira) [inline subgraphs/venus/schema.graphql:27]: I'm thinking of making those also be Mantissa, following the API changes

File: subgraphs/venus/schema.graphql [hunk 2]
```diff
@@ -20,27 +20,27 @@ Market stores all high level variables for a vToken market
...
+    supplyRateMantissa: BigInt!
     "VToken symbol"
     symbol: String!
     "VToken address"
     id: ID!
     "Borrows in the market"
-    totalBorrows: BigDecimal!
+    treasuryTotalBorrowsMantissa: BigInt!
     "VToken supply. VTokens have 8 decimals"
     totalSupply: BigDecimal!
```
Comment (gleiser-oliveira) [inline subgraphs/venus/schema.graphql:45]: This too

File: subgraphs/venus/schema.graphql [hunk 3]
```diff
@@ -58,7 +58,7 @@ type Market @entity {
     "Timestamp the market was most recently updated"
     blockTimestamp: Int!
     "The history of the markets borrow index return (Think S&P 500)"
-    borrowIndex: BigDecimal!
+    borrowIndexMantissa: BigInt!
     "The factor determining interest that goes to reserves"
     reserveFactor: BigInt!
     "Underlying token price in USD"
```
Comment (gleiser-oliveira) [inline subgraphs/venus/schema.graphql:64]: Should become cents

File: subgraphs/venus/src/operations/update.ts [hunk 4]
```diff
@@ -3,61 +3,60 @@ import { Address, BigInt, Bytes, log } from '@graphprotocol/graph-ts';
...
   marketAddress: Address,
   blockNumber: i32,
   blockTimestamp: i32,
 ): Market => {
-  const marketID = marketAddress.toHexString();
-  let market = Market.load(marketID);
+  const marketId = marketAddress.toHexString();
+  let market = Market.load(marketId) as Market;
   if (market == null) {
     log.debug('[updateMarket] market null: {}, creating...', [marketAddress.toHexString()]);
```
Comment (gleiser-oliveira) [inline subgraphs/venus/src/operations/update.ts:43]: Can be removed

File: subgraphs/venus/src/operations/update.ts [hunk 5]
```diff
@@ -72,27 +71,17 @@ export const updateMarket = (
     const exchangeRateStored = contract.try_exchangeRateStored();
     if (exchangeRateStored.reverted) {
       log.error('***CALL FAILED*** : vBEP20 supplyRatePerBlock() reverted', []);
-      market.exchangeRate = zeroBD;
+      market.exchangeRateMantissa = zeroBigInt32;
```
Comment (gleiser-oliveira) [inline subgraphs/venus/src/operations/update.ts:74]: This log will error the subgraph, and won't reach line 74

File: subgraphs/venus/src/utilities/getBnbPriceInUsd.ts [hunk 6]
```diff
@@ -3,13 +3,13 @@ import { Address, BigDecimal } from '@graphprotocol/graph-ts';
...
-import { mantissaFactorBD } from './exponentToBigDecimal';
+import { mantissaFactorBigDecimal } from '../constants';
 
 export function getBnbPriceInUsd(): BigDecimal {
   const comptroller = getOrCreateComptroller();
 
   const oracleAddress = Address.fromBytes(comptroller.priceOracle);
-  const oracle = PriceOracle.bind(oracleAddress);
-  const bnbPriceInUSD = oracle.getUnderlyingPrice(vBnbAddress).toBigDecimal().div(mantissaFactorBD);
+const oracle = PriceOracle.bind(oracleAddress);
```
Comment (gleiser-oliveira) [inline subgraphs/venus/src/utilities/getBnbPriceInUsd.ts:12]: nit: formatting slightly off

File: subgraphs/venus/schema.graphql (hunk 1 above)
Comment (coreyar) [reply subgraphs/venus/schema.graphql:27]: This is not a mantissa

File: subgraphs/venus/src/operations/update.ts (hunk 5 above)
Comment (coreyar) [reply subgraphs/venus/src/operations/update.ts:74]: log.critical will terminate indexing but error won't

**Review by gleiser-oliveira — APPROVED** [review]

**Pull request merged**